src/components/implementation/pong/pingpong/pong.c

```diff
@@ -0,0 +1,24 @@
+#include <cos_component.h>
+#include <cos_kernel_api.h>
+#include <llprint.h>
+#include <pong.h>
+#include <cos_debug.h>
+#include <cos_types.h>
+
+enum {
+	BOOT_SINV_CAP = round_up_to_pow2(BOOT_CAPTBL_FREE + CAP32B_IDSZ, CAPMAX_ENTRY_SZ)
+};
```

Comment:
this `enum` is defined at many places. instead move it to a header and include it wherever required.